main.go

```diff
@@ -38,6 +38,11 @@ func main() {
 		StatFn:  domain.StatFromContext,
 		Fetcher: dbStorage,
 	}
+	fetchAllByTime := &v1.CloudFetchAllByTimestampHandler{
```

Comment:
question: why not call this `fetchAssetsByTime` to match the servefull function name here https://github.com/asecurityteam/asset-inventory-api/pull/29/files#diff-7ddfb3e035b42cd70649cc33393fe32cR62

Reply:
Does `fetchAllAssetsByTime` sound like a better name? (I am trying to convey this does bulk vs. finding a single thing)

Reply:
Yes I like that name, especially using `assets` versus `asset` to distinguish its looking for multiple instead of a singular asset.

Reply:
Changing.